packages/@cruise-automation/hooks/rollup.config.js

```diff
@@ -0,0 +1,10 @@
+//  Copyright (c) 2018-present, GM Cruise LLC
+//
+//  This source code is licensed under the Apache License, Version 2.0,
+//  found in the LICENSE file in the root directory of this source tree.
+//  You may not use this file except in compliance with the License.
+
+import getSharedRollupConfig from "../../../package-shared/getSharedRollupConfig";
```

Comment:
what about calling this directory `common` or `config`?

also, `get` seems uncommon in JS even for functions like this. consider calling it just `sharedRollupConfig`

Reply:
Good idea. I changed `package-shared/getSharedRollupConfig` to `common/rollup.config.js`